pycaret/containers/models/time_series.py

```diff
@@ -240,7 +245,10 @@ def __init__(self, globals_dict: dict) -> None:
             "degree": [1,2,3,4,5],
             "with_intercept": [True, False]
         }
-        tune_distributions = {}
+        tune_distributions = {
+            "degree": randint(low=1, high=10),
```

Comment:
It would be best if you used PyCaret's distributions (pycaret/internal/distributions.py) - feel free to expand upon them if necessary

Reply:
Hi @Yard1 , Any particular reason why? I looked at that before implementing this, but it looks like that is needed only because you are interfacing with many external GS libraries and methods. In this case, it is all internal since it needs custom argument ordering (y, X vs X, y), etc so it wont work with something like BayesGridSearchCV from skoptimize unless we implement out own version custom for TS.

Do you see any issues leaving it as is?

Reply:
I would want to keep it consistent so that it will be easier when we implement wrappers for search algos in the future. Distributions would be the same, after all.

Reply:
@Yard1 , I have implemented the changes to use PyCaret distributions. Please have a look